pkgs/applications/office/flexibee/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  pname = "flexibee";
+  majorversion = "2019.2";
+  version = "${majorversion}.5";
```

Comment:
Generally I like to write this like so:
```
let
version = "2019.2.5";
majorVersion = builtins.substring 0 6 version;
```

I think bots would have an easier time doing updates if they only needed to update 1 variable instead of 2.